### PR TITLE
[BEAM-1360] Remove python-sdk links from the sdks/python/README.md

### PR DESCRIPTION
**Pull request by aaltay**

### CODE REVIEW

**Comment by aaltay** [comment]
R: @prabeesh

**Comment by dhalperi** [comment]
(lgtm)

File: sdks/python/README.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ directory you created.
 
 ### Execute a pipeline locally
 
-The Apache Beam [examples](https://github.com/apache/beam/tree/python-sdk/sdks/python/apache_beam/examples) directory has many examples. All examples can be run locally by passing the arguments required by the example script.
+The Apache Beam [examples](https://github.com/apache/beam/tree/blob/master/sdks/sdks/python/apache_beam/examples) directory has many examples. All examples can be run locally by passing the arguments required by the example script.
```
Comment (prabeesh) [inline sdks/python/README.md]: @aaltay please ensure the updated link is working.

Comment (aaltay) [inline sdks/python/README.md]: Thank you for catching this! Something went wrong with my replace I guess. Fixed this and verified the other links I updated.

PTAL.

**Comment by aaltay** [comment]
Thank you @prabeesh . PTAL.

**Comment by prabeesh** [comment]
LGTM

**Comment by aaltay** [comment]
Thank you @prabeesh. @dhalperi could you merge this?

**Pull request closed without merging**